e2e/README.md

```diff
@@ -0,0 +1,8 @@
+to run e2e tests:
+
+>$ yarn start react-native start
+
```

Comment:
i cant run these e2e tests on my setup, i suspect there is a yarn detox build step maybe?
i will post when i have it up and running for more info

Reply:
ok I got this running by:
1. yarn detox build --configuration android.att.debug
2. start the Pixel_6_API_30 in the emulator
3. yarn detox test --configuration android.att.debug

Reply:
yarn detox test --configuration android.emu.debug
seems to run step 2 and 3 in one step ----^

Reply:
please atleast add this line before line 6 (yarn test):
yarn detox build --configuration android.att.debug